homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -139,6 +140,7 @@ def extra_state_attributes(self):
             ATTR_HOUSE_NUMBER: self._house_number,
             ATTR_POSTCODE: self._postcode,
             ATTR_CITY: self._city,
+            ATTR_STATION_ID: self._station_id,
```

Comment:
What's the use case for the station id in the state attributes?

Reply:
Station_id is a uniqe id per station.
In Api you can filter per radius around one location or you can define the monitored station in cinfigure.yaml.
You got the station_id only via api, not via website.
so if you would define the station via station_id you must grab it from the api. theres no other chance.

so if the station_id is visible via ui then you can  easy get it from there and can modify your configuration.yaml

Reply:
@Delta1977 you can use the "tankstellen finder" webpage (https://creativecommons.tankerkoenig.de/TankstellenFinder/index.html), mentioned in the tankerkönig integration documentation to get the station ID. Without the need of the API.

Reply:
ahh. i need new glasses. 😃 
So this can be closed.
Thank you for the hint